src/drivers/adc/adc.h

```diff
@@ -0,0 +1,25 @@
+#ifndef ADC_H
+#define ADC_H
+
+#include <stdbool.h>
+
+typedef enum {
+    PROCESSOR_BOARD_VOLTAGE = 0,
+    ADC_CHANNEL_2,
+    ADC_CHANNEL_3,
+    ADC_CHANNEL_4,
+    ADC_CHANNEL_5
+} adc_channel_t;

```

Comment:
we should hide these unused channel from public view since it gives a false promise that they'd actually work.